### PR TITLE
Updating alt text for images in Intro-Vector-R: 00-open-a-shapefile

### PR DESCRIPTION
**Pull request by mayastahl**

### CODE REVIEW

**Comment by mayastahl** [comment]
For some reason, a different github account of mine also joined and worked on this. I am going to commit and re-push, but I have to take this account off of this branch first.

**Comment by mayastahl** [comment]
There were some issues with a different GitHub account I had, but it has now been resolved. This should now be an edit to the Intro-Vector-R-00, updating alt image text.

File: tutorials/R/Geospatial-skills/intro-vector-r/00-open-a-shapefile/00-open-a-shapefile.Rmd [hunk 1]
```diff
@@ -85,7 +85,7 @@ with: point, line or polygon.
 
 <figure>
     <a href="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/dc-spatial-vector/pnt_line_poly.png">
-    <img src="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/dc-spatial-vector/pnt_line_poly.png"></a>
+    <img src="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/dc-spatial-vector/pnt_line_poly.png"> alt = "A figure detailing the three different types of vector objects: points, lines, and polygons." </a>
```
Comment (donal-at-NEON) [inline tutorials/R/Geospatial-skills/intro-vector-r/00-open-a-shapefile/00-open-a-shapefile.Rmd:88]: Two notes here:

1.  the alt="..." must be within the closing bracket of the  tag (i.e., move the ">" to after the alt="..." argument)
2. I didn't know this until this afternoon, but alt text is best when it doesn't begin with "A figure.." or "A diagram..". I just added a new link to the section on Alt text (https://www2.le.ac.uk/webcentre/plone/build/basics/add-images/alt-text). Perhaps you could just do: "The three different types of vector objects: points, lines, and polygons"

Suggest:
` 
</a> `
with the closing tag `</a>` on a new line.

File: tutorials/R/Geospatial-skills/intro-vector-r/00-open-a-shapefile/00-open-a-shapefile.Rmd [hunk 2]
```diff
@@ -229,7 +229,7 @@ in the CRS **UTM zone 18N**. The CRS is critical to interpreting the object
 
 <figure>
     <a href="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/dc-spatial-vector/spatial_extent.png">
-    <img src="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/dc-spatial-vector/spatial_extent.png"></a>
+    <img src="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/dc-spatial-vector/spatial_extent.png"> alt="A figure depicting how different vector types are represented within a given spatial extent." </a>
```
Comment (donal-at-NEON) [inline tutorials/R/Geospatial-skills/intro-vector-r/00-open-a-shapefile/00-open-a-shapefile.Rmd:232]: Same as above

File: tutorials/R/Geospatial-skills/intro-vector-r/00-open-a-shapefile/00-open-a-shapefile.Rmd [hunk 3]
```diff
@@ -250,7 +250,7 @@ attributes stored with it.
 
 <figure>
     <a href="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/dc-spatial-vector/Attribute_Table.png">
-    <img src="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/dc-spatial-vector/Attribute_Table.png"></a>
+    <img src="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/dc-spatial-vector/Attribute_Table.png"> alt="A figure depicting example attribute tables for each different type of vector object."</a>
```
Comment (donal-at-NEON) [inline tutorials/R/Geospatial-skills/intro-vector-r/00-open-a-shapefile/00-open-a-shapefile.Rmd:253]: There's an old motto in the education literature that, to remember any new skill, people have to practice it three times.. :)

**Comment by mayastahl** [comment]
@donal-at-NEON Looks like it went through this time. Still learning the process for this but it's pretty cool!

**Pull request merged**